### DISCUSSION TITLE
Duo free plan possible?

### DISCUSSION THREAD

**Discussion by bbrendon** [neos-sdi/adfsmfa#213] [Q&A] [answered]
I have a personal DUO free plan and currently they only support ADFS with their paid version. Is there some way to use this with their free plan?

**Comment by redhook62** [comment]
Frankly, I don't know at all. best to contact them directly.

regards

**Comment by bbrendon** [comment, marked as answer]
Thanks. if you ever add support for duo plz update this issue and let me
know.
